### PR TITLE
New Nginx Tracing Module

### PR DESCRIPTION
**Pull request by dgoffredo**
### What does this PR do?
This revision adds onboarding documentation for the new nginx module.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_collection/proxy_setup/_index.md
Comment (hestonhoffman) [inline content/en/tracing/trace_collection/proxy_setup/_index.md:333]:
```suggestion
In the topmost section of the Nginx configuration, load the Datadog module.
```

File: content/en/tracing/trace_collection/proxy_setup/_index.md
Comment (dgoffredo) [reply content/en/tracing/trace_collection/proxy_setup/_index.md:749]: Ah, that explains it.  Thanks.

File: content/en/tracing/trace_collection/proxy_setup/_index.md
Comment (yoannrobin) [inline content/en/tracing/trace_collection/proxy_setup/_index.md:348]: Is service part of the documentation? (https://github.com/DataDog/nginx-datadog/blob/master/doc/API.md#datadog)

File: content/en/tracing/trace_collection/proxy_setup/_index.md
Comment (yoannrobin) [inline content/en/tracing/trace_collection/proxy_setup/_index.md:342]:
```suggestion
As an example, to set a service name to `usage-internal-nginx` and sample 10% of your traces:
```

File: content/en/tracing/trace_collection/proxy_setup/_index.md
Comment (yoannrobin) [inline content/en/tracing/trace_collection/proxy_setup/_index.md:464]: Does it also apply to Nginx module? As this section falls under the Nginx with opentracing module I would assume it is only relevant for this case

File: content/en/tracing/trace_collection/proxy_setup/_index.md
Comment (yoannrobin) [inline content/en/tracing/trace_collection/proxy_setup/_index.md:357]: Should we add the evaluation order between directive and environment variables?

File: content/en/tracing/trace_collection/proxy_setup/_index.md
Comment (dgoffredo) [reply content/en/tracing/trace_collection/proxy_setup/_index.md:348]: yes

File: content/en/tracing/trace_collection/proxy_setup/_index.md
Comment (dgoffredo) [reply content/en/tracing/trace_collection/proxy_setup/_index.md:464]: `### Nginx Sampling` is an `h3` section, the same as the OpenTracing and Datadog module sections.  So, there is one section about getting the Datadog module, another about getting the OpenTracing module, and then a third about configuring sampling for them.

This is possibly confusing.  What might be better?

Comment (dgoffredo) [reply content/en/tracing/trace_collection/proxy_setup/_index.md:464]: I am wrong.  Will fix.

File: content/en/tracing/trace_collection/proxy_setup/_index.md
Comment (dgoffredo) [reply content/en/tracing/trace_collection/proxy_setup/_index.md:357]: That's covered in the documentation of the environment variables, which is linked in the API documentation.

File: content/en/tracing/trace_collection/proxy_setup/_index.md
Comment (yoannrobin) [inline content/en/tracing/trace_collection/proxy_setup/_index.md:308]:
```suggestion
<div class="alert alert-warning">
Datadog Nginx module is currently in beta. <a href="https://docs.datadoghq.com/help/">Reach out to support</a> if you have any feedback.
</div>
```

**Review by yoannrobin — APPROVED** [review]

**Comment by dgoffredo** [comment]
This is ready for editorial review.

Thanks for the comments everyone so far.

**Comment by dgoffredo** [comment]
Hey @hestonhoffman, are you a code owner who can approve and merge this PR, or should I @ someone else?

**Comment by hestonhoffman** [comment]
@dgoffredo Yes! I'll take a quick look after lunch and approve/merge

**Review by hestonhoffman — APPROVED** [review]

**Comment by dgoffredo** [comment]
Thanks!

**Pull request merged**
